configure.ac

```diff
@@ -1622,6 +1622,8 @@ AC_MSG_RESULT($build_bitcoin_libs)
 
 AC_LANG_POP
 
+CC_UNPREFIXED=$CC
+AC_SUBST(CC_UNPREFIXED)
```

Comment:
Why is this needed?

Reply:
To enable `ccache` the code below in `configure.ac` makes the `CC` variable looks like that
```
<path_to_ccache> <path_to_compiler>
```
Then https://github.com/bitcoin/bitcoin/blob/5c041cb348e667672c8b5ed504dba05adf60783b/src/Makefile.qt.include#L390

the `make` function `$(dir $(CC))` treats the `CC` variable as a list which consist of two paths, and, eventually, the `cp` command fails.